static/sass/main.scss

```
@@ -0,0 +1,13 @@
+$main-color: #293462; //rgb(41, 52, 98)
+$main-color-dark: #565D90; //rgb(86, 93, 144)
+$main-color-light: #000E37; //rgb(0, 14, 55)
+
+$complementary-color: #216583; //rgb(33, 101, 131)
```

Naming conventions for colors are hard, and I would be fine leaving this as-is. However, I think a clearer color naming scheme might be something like this: 

```suggestion
$brand-color--dark-blue: #293462; 
$brand-color--purple-blue: #565D90; 
$brand-color--midnight-blue: #000E37; 
$brand-color--light-blue: #216583; 
$accent-color--red: ##ff4231;
$accent-color--orange: #ff9119; 
$accent-color--gray: ##f2f2f4;
```

You did have the `$main-color-dark` and the `$main-color-light`, so those will need to be switched if you keep the naming scheme you have. 

Also, unless there is a reason they are helpful to have, I would remove the comments with the RGB values.

---

this is labeled as outdated as well. We can implement these changes if needed. But at the end the 4 variables of materialize

$primary-color: 
$primary-color-light: 
$primary-color-dark:
$secondary-color:

need to be set to our defined color variables for materialize to work with those colors